lib/fizzy/parser_expr.cpp

```diff
@@ -122,9 +123,11 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, const Mod
     // For a block/if/else instruction the value is the block/if/else's immediate offset.
     std::stack<ControlFrame> control_stack;
 
-    // TODO: Get function result arity from the function type in Module.
-    const uint8_t func_result_arity = 0;
-    control_stack.emplace(Instr::block, func_result_arity, 0);  // The function's implicit block.
+    const auto func_type_idx = module.funcsec[func_idx];
```

Comment:
Why not pass in `func_type_idx`?

Reply:
It was like that originally, I changed it just because `func_idx` seems to be a more natural parameter to "parse instuctions of a function", might be more useful in the future for something.